.github/workflows/tests.yaml

```
@@ -706,6 +706,61 @@ jobs:
         name: tests-report-e2e
         path: e2e-tests.xml
 
+  e2e-tests-chainsaw:
+    runs-on: ubuntu-latest
+    needs:
+    - check-docs-only
+    - matrix_k8s_node_versions
+    if: ${{ needs.check-docs-only.outputs.docs_only != 'true' }}
+    env:
+      IMG: kong-operator
+      TAG: e2e-${{ github.sha }}
+      CLUSTER_NAME: e2e-chainsaw
+    steps:
+    - name: Harden Runner
+      uses: step-security/harden-runner@df199fb7be9f65074067a9eb93f12bb4c5547cf2 # v2.13.3
+      with:
+        egress-policy: audit
+    - name: checkout repository
+      uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # v6.0.1
+
+    - name: setup golang
+      uses: actions/setup-go@4dc6199c7b1a012772edbd06daecab0f50c9053c # v6.1.0
+      with:
+        go-version-file: go.mod
+
+    - name: build docker image
+      run: make docker.build
+
+    - uses: jdx/mise-action@146a28175021df8ca24f8ee1828cc2a60f980bd5 # v3.5.1
+      with:
+        install: false
+
+    - name: Create k8s KinD Cluster with ktf
+      run: |
+        go install github.com/kong/kubernetes-testing-framework/cmd/ktf@latest
+        ktf environments create --name ${{ env.CLUSTER_NAME }} --addon metallb --addon cert-manager
+
+    - name: Load image to kind cluster
+      run: kind load docker-image ${{ env.IMG }}:${{ env.TAG }} --name ${{ env.CLUSTER_NAME }}
+
+    - name: Deploy operator with Helm
+      run: |
+        helm install kong-operator charts/kong-operator \
```

```suggestion
        helm install kong-operator ./charts/kong-operator \
```

Don't we want to use the in tree chart?

---

Good catch!

---

By the way, using the in-tree chart explicitly makes things clearer.

Before this change, we were in fact already using the in-tree chart. :smile: